www/core/nightlies/next_patch_list.xml

```diff
@@ -3,7 +3,7 @@
 	<extension name="Joomla" element="joomla" type="file" version="3.10.12-dev" targetplatformversion="3.8" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.10.12-dev" targetplatformversion="3.9" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
 	<extension name="Joomla" element="joomla" type="file" version="3.10.12-dev" targetplatformversion="3.10" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.2.1-dev" targetplatformversion="4.0" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.2.1-dev" targetplatformversion="4.1" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
-	<extension name="Joomla" element="joomla" type="file" version="4.2.1-dev" targetplatformversion="4.2" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
+	<extension name="Joomla" element="joomla" type="file" version="4.2.2-dev" targetplatformversion="4.0" detailsurl="https://update.joomla.org/core/nightlies/next_patch_extension.xml" />
```

Comment:
@zero-24 Here they should also go to 4.0.4 first or not? since they are coming from 4.0

Reply:
That are just the nightly builds. There it is expected that might something break we also do not handle other fall back steps within that update server to keep it as simple as possible.
So yes you are right that we do not handle it there but deliberate to keep it simple. In this case it might make sense to add that fallback there too. Will prepare it and send an update to this PR once done.

Reply:
Done within the RC update server PR will be merged in here once the other PR has been merged.